app/Http/Middleware/CacheControl.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Ushahidi\App\Http\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Auth\Factory as Auth;
+
+class CacheControl
+{
+    /**
+     * The authentication guard factory instance.
+     *
+     * @var \Illuminate\Contracts\Auth\Factory
+     */
+    protected $auth;
+
+    // TODO: make these configurable without modifying source code
+    private $policies = [
+        'performance' => [ 'max-age' => 'max-age=600' ],
+        'conservative' => [ 'max-age' => 'max-age=10' ],
+        'forbid-caching' => [ 'cache' => 'no-store' ]
+    ];
+
+    /**
+     * Create a new middleware instance.
+     *
+     * @param  \Illuminate\Contracts\Auth\Factory  $auth
+     * @return void
+     */
+    public function __construct(Auth $auth)
+    {
+        $this->auth = $auth;
+    }
+
+    protected function isAuthorized(string $guard)
+    {
+        return !($this->auth->guard($guard)->guest());
+    }
+
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  string $policy  selected general policy
+     * @param  string $exceptAuthGuard
+     *                don't apply to requests satisfying given authentication guard
+     * @param  string $privateIfAuthGuard
```

Comment:
are we expecting a bool here? (or am I just confused)

Reply:
no, a string, it's just like the Auth middleware: 
https://github.com/ushahidi/platform/blob/develop/app/Http/Middleware/Authenticate.php#L33
as seen used here:
https://github.com/ushahidi/platform/blob/develop/routes/config.php#L17

Kind of a boolean really since I think we would always use "api" or leave empty, but I just didn't want to introduce another pattern of usage.